week-5/A-dom-manipulation/exercise.js

```diff
@@ -4,7 +4,24 @@ Task 1
 
 Write JavaScript below that changes the background colour of the page when the 'Change colour' button is clicked.
 */
+////////////METHOD 1//////
+// var bgrChangeBtn = document.querySelector("#bgrChangeBtn");
+// bgrChangeBtn.addEventListener("click", changeBackground);
 
+// function changeBackground() {
+//     document.body.style.backgroundColor = "red";
+
+// }
+
+//////////METHOD 2: CREATING AN ARRAY WITH THE CHILDREN OF THE DIV//////////
+var divButtonChild = document.querySelector(".buttons").children;
+//console.log(divButtonChild);
```

Comment:
Minor point: This is an array, so even if you think it's going to have one element in it, it should be called `divButtonChildren`, not `...Child`.